CONTRIBUTING.md

```diff
@@ -10,9 +10,32 @@ If you've found a bug, please report it to the issue tracker and
 
 ### Development
 
-#### Installation using pip
+#### Setup
 
-To get started, set up a new virtual environment and install all requirements
+##### Installation using conda
+
+To get started with conda, install conda according to the official guide, set up a conda environment and install all
+requirements.
+
+```bash
+conda create --name conda_kartothek python=3.6
+conda activate conda_kartothek
+conda install pip
+pip install -e .
+pip install -r test-requirements.txt
```

Comment:
Conda install cannot parse the current iteration of our requirements.txt file, as it cannot deal with:
"package; python `>` `<` `=` version"

This was basically meant as setting up with conda in a "compatibililty mode". Primarily to ease the user into using conda in order to be able to build the docs later.

It may be not the intended conda workflow but it is and will be working going forward.

Disclaimer: I am a "pip guy" and just wanted to get the docs working yesterday. I have to say that I'm not quite happy with the state of mixing the requirement for pip and conda in the contribution workflow myself. But I guess this is the current workflow (please correct me if I'm wrong).